tapir/accounts/templates/accounts/user_detail.html

```diff
@@ -163,7 +161,7 @@ <h5 class="card-header d-flex justify-content-between align-items-center flex-wr
     </div>
     <div class="row">
         <div class="col-xl-4">{% purchase_tracking_card tapir_user=object %}</div>
-        {% if tapir_user.share_owner and tapir_user.allows_purchase_tracking %}
+        {% if tapir_user.share_owner and tapir_user.allows_purchase_tracking and object.pk == request.user.pk or perms.coop.admin %}
```

Comment:
If the user disabled tracking, the admins can still see the purchase data. Should it be hidden?